front-end/src/App.css

```diff
@@ -10,4 +46,12 @@
   transition: all ease .5s;
   background-color: red;
   color: white;
+}
+
+.sound-clip-container {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 50%;
```

Comment:
Itty bitty gritty: Setting width inside a flex container works in most browsers, but flex-basis would probably be better. It officially accounts for margin/padding when calculating size.